roboteam_ai/src/conditions/HasBall.cpp

```diff
@@ -25,7 +25,7 @@ bt::Node::Status HasBall::update() {
     if (! robot || ! ball) {
         return Status::Failure;
     }
-    if (World::robotHasBall(*robot, *ball)) {
+    if (World::ourBotHasBall(robot->id)) {
```

Comment:
I would suggest robotHasBall(robot->id, bool ourTeam)

Reply:
agreed.

Reply:
Done

Reply:
No you still have ourBotHasBall and theirBotHasBall now

Reply:
should be fixed

Reply:
Yes, this